bin/install

```diff
@@ -244,11 +244,13 @@ EOF
 
     # stream package file to disk
     begin
+      retries ||= 0
       uri.open(:ssl_verify_mode => OpenSSL::SSL::VERIFY_PEER, :redirect => true, :read_timeout => 120, :proxy => @http_proxy) do |s3|
         package_file.write(s3.read)
       end
     rescue OpenURI::HTTPError => e
-      @log.error("Could not find package to download at '#{uri.to_s}'")
+      retry if (retries +=1) < 5
```

Comment:
I think adding a retry here is a good idea. However, I think we might want to address a couple considerations:

1. The error that you mention in your ticket is a `OpenSSL::SSL::SSLError`, which would not be handled here. It will just jump out of here anyway, so we probably want to add that to address the specific case.
2. Retries are a good addition to this call, but it would be best if we added exponential backoff to the retry to get the best results, which will give us more opportunities for success in cause the issue lasts for 5 rapid fire calls.